pkg/controller/controller_instance_test.go

```diff
@@ -115,42 +115,34 @@ func TestReconcileInstanceWithAuthError(t *testing.T) {
 
 	testController.reconcileInstance(instance)
 
+	// verify that one catalog client action occurred
 	actions := fakeCatalogClient.Actions()
-	assertNumberOfActions(t, actions, 1)
```

Comment:
Not sure which current methods you're referring to. This and the below removed lines were all replaced by the single call to `checkCatalogClientActions` below